src/OrleansRuntime/Http/ExtensionMethods.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Owin;
+using Newtonsoft.Json;
+using System;
+using System.Threading.Tasks;
+
+namespace Orleans.Runtime
+{
+    internal static class ExtensionMethods
+    {
+        public static Task ReturnJson(this IOwinContext context, object value)
```

Comment:
I'm not opposed to moving the HTTP pieces to a separate assembly, I was following the advice here: https://github.com/dotnet/orleans/issues/2427#issuecomment-263644610